data/text/common_2.asm

```diff
@@ -340,274 +340,275 @@ _AskSurfText::
 	line "Want to SURF?"
 	done
 
-UnknownText_0x1c068e::
+_UseWaterfallText::
 	text_ram wStringBuffer2
 	text " used"
 	line "WATERFALL!"
 	done
 
-UnknownText_0x1c06a3::
+_BigWaterfallText::
```

Comment:
_HugeWaterfallText (...really, why "big" instead of "huge" when it's in the text?)